### PR TITLE
Add support for Wacom Bamboo Pad USB CTH-301 tablets

### PR DESCRIPTION
**Pull request by aomsin2526**
This thing is different from any other wacom tablets, so new parser is needed. Pen, eraser, pen and aux button works. No touch support yet.

### CODE REVIEW

**Comment by jamesbt365** [comment]
closes #2312

**Comment by 32-bit-link** [comment]
Thank you aom sin for this magnificent pull request 🙏

File: OpenTabletDriver.Configurations/Parsers/Wacom/BambooPad/BambooPadAuxReport.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public struct BambooPadAuxReport : IAuxReport
+    {
+        public BambooPadAuxReport(byte[] report)
+        {
+            Raw = report;
+
+            AuxButtons = new bool[]
+            {
+                report[23] == 1,
+                report[23] == 2
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Parsers/Wacom/BambooPad/BambooPadAuxReport.cs:16]: This prevents it from sending both buttons as active. Is it not possible for the hardware to send both?

```suggestion
                report[23].IsBitSet(0),
                report[23].IsBitSet(1)
```

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Configurations/Parsers/Wacom/BambooPad/BambooPadAuxReport.cs (hunk 1 above)
Comment (aomsin2526) [reply OpenTabletDriver.Configurations/Parsers/Wacom/BambooPad/BambooPadAuxReport.cs:16]: As far as i tested, This value is only returns 1 or 2, It is not possible to click both buttons as same time even on official drivers. It is also prioritize right button.

Comment (aomsin2526) [reply OpenTabletDriver.Configurations/Parsers/Wacom/BambooPad/BambooPadAuxReport.cs:16]: After another test i found problem related to long press, maybe i will try improve it later.

**Review by X9VoiD — APPROVED** [review]

**Pull request merged**